File: Q08/main.cpp
#include<iostream>
using namespace std;

int main(){
    int row,col;
    cin>>row>>col;

    int arr[row][col];

    for(int i=0;i<row;i++){
        for(int j=0;j<col;j++){
            cin>>arr[i][j];
        }
    }

    int min = arr[0][0];
    for(int i=0;i<row;i++){
        for(int j=0;j<col;j++){
            if(arr[i][j]<min){
                min = arr[i][j];
            }
        }
    }
    cout<<min;
    return 0;
}

File: Q08/main.py
row = int(input())
col = int(input())

arr=[]

for i in range(row):
    arr.append(list(map(int,input().split()))[:col])

min = arr[0][0]

for i in range(row):
    for j in range(col):
        if arr[i][j]<min:
            min=arr[i][j]

print(min)